diff --git a/class-parser.php b/class-parser.php
index 4b13dc1..2d78de2 100644
--- a/class-parser.php
+++ b/class-parser.php
@@ -714,7 +714,7 @@ class Parser {

 			// In the event that something invalid is used, we'll ignore it (Example: 'Joe Bloggs (Australian Translation)')
 			if ( ! $user ) {
-				$this->warnings['contributor_ignored'] ??= [];
+				$this->warnings['contributor_ignored'] = isset( $this->warnings['contributor_ignored'] ) ? $this->warnings['contributor_ignored'] : [];
 				$this->warnings['contributor_ignored'][] = $name;
 				unset( $users[ $i ] );
 				continue;
